rapid_pro/docker-run-synchronise-contacts.sh

```diff
@@ -23,23 +23,23 @@ done
 # Check that the correct number of arguments were provided.
 if [[ $# -ne 5 ]]; then
     echo "Usage: ./docker-run.sh
-    <google-cloud-credentials-file-path> <source-domain> <source-credentials-url>
-    <target-domain> <target-credentials-url>"
+    <google-cloud-credentials-file-path> <instance-1-domain> <instance-1-credentials-url>
+    <instance-2-domain> <instance-2-credentials-url>"
     exit
 fi
 
 # Assign the program arguments to bash variables.
 GOOGLE_CLOUD_CREDENTIALS_FILE_PATH=$1
-SOURCE_DOMAIN=$2
-SOURCE_CREDENTIALS_URL=$3
-TARGET_DOMAIN=$4
-TARGET_CREDENTIALS_URL=$5
+INSTANCE_1_DOMAIN=$2
```

Comment:
```suggestion
WORKSPACE_1_DOMAIN=$2
```

Reply:
hmm good point, TextIt calls these workspaces. I'll update this and RapidProTools appropriately.

Reply:
Renamed here and in many other places in this tools subdirectory